Berksfile

```diff
@@ -1,10 +1,10 @@
-source 'https://supermarket.getchef.com'
+source 'https://supermarket.chef.io'
 
 metadata
 
 group :integration do
-  cookbook 'test', :path => './test/fixtures/cookbooks/test'
+  cookbook 'test', :path => 'test/fixtures/cookbooks/test'
 end
 
 # This cookbook isn't on supermarket.
-cookbook 'chef-server-ingredient', github: 'opscode-cookbooks/chef-server-ingredient'
+cookbook 'chef-server-ingredient', github: 'stephenlauck/chef-server-ingredient', branch: 'reconfigure_parameter_for_install'
```

Comment:
If the change in your branch is required we should get that merged and continue to point to the opscode-cookbooks upstream.